ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImagesTable.tsx

```diff
@@ -0,0 +1,172 @@
+                    operatingSystem,
+                    deploymentCount,
+                    metadata,
+                    watchStatus,
+                    scanTime,
+                }) => {
+                    return (
+                        <Tbody
+                            key={id}
+                            style={{
+                                borderBottom: '1px solid var(--pf-c-table--BorderColor)',
+                            }}
+                        >
+                            <Tr>
+                                <Td>
+                                    {name ? (
+                                        <ImageNameTd name={name} id={id} />
+                                    ) : (
+                                        'Image name not available'
+                                    )}
+                                </Td>
+                                <Td>
+                                    <SeverityCountLabels
+                                        critical={imageCVECountBySeverity.critical}
+                                        important={imageCVECountBySeverity.important}
+                                        moderate={imageCVECountBySeverity.moderate}
+                                        low={imageCVECountBySeverity.low}
+                                    />
+                                </Td>
+                                <Td>{operatingSystem}</Td>
+                                <Td>
+                                    {/* TODO: add modal */}
+                                    {deploymentCount > 0 ? (
+                                        <Button
+                                            variant={ButtonVariant.link}
+                                            isInline
+                                            component={LinkShim}
+                                            href={getEntityPagePath('Deployment', id)}
+                                        >
+                                            {deploymentCount}{' '}
+                                            {pluralize('deployment', deploymentCount)}
+                                        </Button>
+                                    ) : (
+                                        <Flex>
+                                            <div>0 deployments</div>
+                                            {/* TODO: double check on what this links to */}
+                                            <span>({`${watchStatus}`} image)</span>
+                                        </Flex>
+                                    )}
+                                </Td>
+                                <Td>
+                                    <Tooltip content={getDateTime(metadata?.v1?.created)}>
+                                        <div>
+                                            {getDistanceStrictAsPhrase(
+                                                metadata?.v1?.created,
```

Comment:
We should test and see if this blows up if `metadata?.v1?.created` ends up as `undefined` - it looks like our typing of that function is incorrect since it currently accepts `any` as a parameter.

Reply:
if i replace `metadata?.v1?.created` with `undefined`, i get `NaN years ago` which isn't the best but also isn't the worst. i'll follow up w the team to see what would be a good replacement here and send it as a separate PR.